packages/libs/src/sdk/api/albums/AlbumsApi.ts

```diff
@@ -214,4 +229,179 @@ export class AlbumsApi {
       advancedOptions
     )
   }
+
+  /**
+   * Purchases stream access to an album
+   *
+   * @hidden
+   */
+  async purchase(params: PurchaseAlbumRequest) {
+    const {
+      userId,
+      albumId,
+      price: priceNumber,
+      extraAmount: extraAmountNumber = 0,
+      walletAdapter
+    } = await parseParams('purchase', PurchaseAlbumSchema)(params)
+
+    const contentType = 'album'
+    const mint = 'USDC'
+
+    // Fetch album
+    this.logger.debug('Fetching album...', { albumId })
+    const { data: albums } = await this.getAlbum({
+      userId: params.userId, // not sure why this is required
```

Comment:
It should be optional...

https://github.com/AudiusProject/audius-protocol/blob/544839db86f758fe709b5c5b7b9c4fc13de66ea4/packages/libs/src/sdk/api/albums/types.ts#L11

https://github.com/AudiusProject/audius-protocol/blob/544839db86f758fe709b5c5b7b9c4fc13de66ea4/packages/libs/src/sdk/api/generated/default/apis/PlaylistsApi.ts#L37

it calls playlist under da hood